src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo.type';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { completed, id, title, user } = todo;
+
+  return (
+    <article
+      key={id}
```

Comment:
The `key` attribute is not needed here as this component is not part of a list. The `key` attribute is used by React to identify elements in a list for efficient updates. You can safely remove it from this component.